src/utils/transactions/TransactionResultUtil.ts

```diff
@@ -288,15 +298,62 @@ export const EcommerceMaybeInterruptStatusCodeEnumType =
     "EcommerceMaybeInterruptStatusCodeEnumType"
   );
 
+/**
+ * This function will match any status code from closePaymentResultError with any
+ * status code defined in the ClosePaymentErrorsMap item.
+ *
+ * NOTE:
+ * ClosePaymentErrorsMap supports placeholders, so 5xx will match any error >= 500
+ */
+function evaluateClosePaymentResultError(
+  nodeInfo?: TransactionInfoNodeInfo
+): ViewOutcomeEnum {
```

Comment:
Is it possible to simplify this method? For example for me using regexes here is too complex and makes the code less readable.

Reply:
I think the method is described by the many comments that comes with it, and the name of the functions are quite straighforward (matchStatusCode is not hard to guess). I don't think that the presence of a regex is a problem.